src/humancellatlas/data/metadata/api.py

```diff
@@ -582,8 +583,11 @@ class ImagingPreparationProtocol(Protocol):
     pass
 
 
-@dataclass
+@dataclass(init=False)
 class ManifestEntry:
+    # FIXME: would be MutableJSON is Azul types were available
+    # https://github.com/DataBiosphere/hca-metadata-api/issues/13
+    json: MutableMapping = field(repr=False)
```

Comment:
Took me a while to notice this: declaring this mutable is dangerous and inconsistent with precedent. Please be more diligent in honouring precedent. If you think a deviation from precedent is warranted, discuss that with the team and implement the deviation consistently.